Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebButtonsView.swift

```diff
@@ -1,49 +1,52 @@
 import SwiftUI
 import PaymentButtons
+import PayPalWebPayments
 
 struct PayPalWebButtonsView: View {
 
     @ObservedObject var payPalWebViewModel: PayPalWebViewModel
 
+    @State private var selectedFundingSource: PayPalWebCheckoutFundingSource = .paypal
+
     var body: some View {
         VStack {
-            VStack(alignment: .center, spacing: 40) {
-                PayPalButton.Representable(color: .blue, size: .mini) {
-                    payPalWebViewModel.paymentButtonTapped(funding: .paypal)
+            VStack(alignment: .center, spacing: 16) {
+                HStack {
+                    Text("Checkout with PayPal")
+                        .font(.system(size: 20))
+                    Spacer()
                 }
-                .frame(maxWidth: .infinity, maxHeight: 40)
-                PayPalCreditButton.Representable(color: .black, edges: .softEdges, size: .expanded) {
-                    payPalWebViewModel.paymentButtonTapped(funding: .paypalCredit)
+                .frame(maxWidth: .infinity)
+                .font(.headline)
+                Picker("Funding Source", selection: $selectedFundingSource) {
+                    Text("PayPal").tag(PayPalWebCheckoutFundingSource.paypal)
+                    Text("PayPal Credit").tag(PayPalWebCheckoutFundingSource.paylater)
+                    Text("Pay Later").tag(PayPalWebCheckoutFundingSource.paypalCredit)
                 }
-                .frame(maxWidth: .infinity, maxHeight: 40)
-                PayPalPayLaterButton.Representable(color: .silver, edges: .rounded, size: .full) {
-                    payPalWebViewModel.paymentButtonTapped(funding: .paylater)
+                .pickerStyle(SegmentedPickerStyle())
+
+                switch selectedFundingSource {
+                case .paypalCredit:
+                    PayPalCreditButton.Representable(color: .black, size: .full) {
```

Comment:
All the buttons are the same size now. We have a separate `Payment Button` demo that displays all of the options of size/color/etc.